### DISCUSSION TITLE
DELETE raises DeprecationWarning: Use 'content=<...>' to upload raw bytes/text content.

### DISCUSSION THREAD

**Discussion by emilt27** [encode/httpx#1705] [Potential Issue]
I use HTTPX in the following way:
```
def delete(*args, **kwargs):
    async with AsyncClient(app=app, base_url=base_url) as async_client:
        headers = kwargs.pop('headers', {})
        headers.update(auth_headers)
        return await async_client.delete(*args, headers=headers, **kwargs)
```

And then:
```
delete('/disconnect')
```

It produces the next warning message:
```
  /usr/local/lib/python3.8/site-packages/httpx/_content.py:173: DeprecationWarning: Use 'content=<...>' to upload raw bytes/text content.
    warnings.warn(message, DeprecationWarning)
```

I use the latest version of HTTPX `httpx==0.18.2`. 
How can I get rid of this warning?

**Comment by lovelydinosaur** [comment]
You've not quite given a complete example here, so it's a bit hard to help out. Perhaps you could try to reduce this into a stand-alone reproducible example?

Using `data=<bytes>` is deprecated in favour of `content=<bytes>`, so *normally* you'd see this warning if you were sending a request using the `data` keyword argument when the `content` keyword argument should be used instead.

**Reply by emilt27** [reply]
@tomchristie I've rechecked it and it looks like the issue is in https://docs.authlib.org/en/latest/client/api.html#authlib.integrations.httpx_client.AsyncOAuth2Client.revoke_token
